### PR TITLE
DA-31-login-user

### PR DESCRIPTION
**Pull request by afekTheMiniLearner**

### CODE REVIEW

File: client/src/components/LoginForm/LoginForm.jsx [hunk 1]
```diff
@@ -66,6 +75,13 @@ export default withFormik({
     const { username, password } = values;
     const data = { username, password };
 
+    AuthProxy.loginUser(data)
```
Comment (hdriel) [inline client/src/components/LoginForm/LoginForm.jsx:78]: as same feedback as RegisterForm component

File: client/src/proxies/proxy.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ export default class BaseProxy {
 
     try {
       const response = await axios.get(requestUrl);
-      return response.data;
+      return response;
```
Comment (hdriel) [inline client/src/proxies/proxy.js:14]: why did you remove the response.data , 
do you expect to anything else? 
if requested done successfully (it's mean status 200) do return his data 
unless you going to catch section 
then expected to parse the error , like: 
`error.response.data.error.message`

File: client/src/proxies/proxy.js [hunk 3]
```diff
@@ -40,8 +40,10 @@ export default class BaseProxy {
     const method = 'POST';
 
     try {
-      const response = await axios.post(requestUrl, data);
-      return response.data;
+      const response = await axios.post(requestUrl, data, {
+        withCredentials: true,
```
Comment (hdriel) [inline client/src/proxies/proxy.js:44]: the credentials include should pass as optional param in ALL requests , 
you have also authenticated request also on get/delete/put method APIs, 
not only on post APIs

File: client/src/screens/Login/Login.jsx [hunk 4]
```diff
@@ -5,20 +5,28 @@ import { Alert, PageContainer, Snackbar } from './Login.styled';
 
 export default function Login() {
   const [userData, setUserData] = useState(null);
-  // Just for demonstration, will be used with API request result
-  const responseCode = 500;
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
 
+  console.log('isLoggedIn', isLoggedIn);
```
Comment (hdriel) [inline client/src/screens/Login/Login.jsx:10]: remove console log please

File: server/src/app.ts [hunk 5]
```diff
@@ -31,7 +31,7 @@ if (process.env.NODE_ENV !== 'test') {
 
 const app = express();
 
-app.use(cors());
+app.use(cors({ credentials: true, origin: 'http://localhost:3030' }));
```
Comment (hdriel) [inline server/src/app.ts:34]: move the origin value into environment variables

File: client/src/components/LoginForm/LoginForm.jsx [hunk 6]
```diff
@@ -14,8 +15,16 @@ import {
...
 
 const LoginForm = (props) => {
-  const { errors, handleBlur, handleChange, handleSubmit, touched, values } =
-    props;
+  const {
+    errors,
+    handleBlur,
+    handleChange,
+    // eslint-disable-next-line no-unused-vars
+    setIsLoggedIn,
```
[the remark this answers is not in the document]
Comment (afekTheMiniLearner) [reply client/src/components/LoginForm/LoginForm.jsx:23]: im using it in the handleSubmit function of formik
